### PR TITLE
TSM fix: now rewarding for Cannibals

### PR DESCRIPTION
**Pull request by Gabriel-SE**
And a few code style changes

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -11051,11 +11051,11 @@ void player::do_read( item *book )
             book->typeId() == "cookbook_human" ) {
             fun_bonus = 25;
             add_morale(MORALE_BOOK, fun_bonus, fun_bonus * 3, 60, 30, true, book->type);
-        } if (has_trait("SPIRITUAL") && book->has_flag("INSPIRATIONAL")) {
-              fun_bonus = 15;
+        } else if ( has_trait("SPIRITUAL") && book->has_flag("INSPIRATIONAL") ) {
+            fun_bonus = 15;
```
Comment (KA101) [inline src/player.cpp]: Uh, this is the religious texts; the cannibal cookbook is the next one up.  I'm pretty sure To Serve Man isn't considered Inspirational, so I'm at a loss for why the need to else this if.

**Comment by Rivet-the-Zombie** [comment]
I must've broken this when I fixed religious books.

**Comment by KA101** [comment]
That's as may be, but I'm not at all sure how this fixes the problem.  TSM isn't INSPIRATIONAL so it shouldn't be triggering the if here.

**Comment by Gabriel-SE** [comment]
If the book is TSM, without else if the first cannibal check will add correct morale bonus but the last else will also be executed because TSM isn't inspirational.
Since the negative morale bonus from else is bigger (tmp->fun * 15), it will end up in negative morale loss.

**Comment by KA101** [comment]
OK, that'll do.  Thanks.

**Pull request merged**